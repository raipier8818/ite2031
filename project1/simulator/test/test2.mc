8454148
4259841
12648452
25165824
7
