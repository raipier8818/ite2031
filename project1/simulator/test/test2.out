memory[0]=8454148
memory[1]=4259841
memory[2]=12648452
memory[3]=25165824
memory[4]=7

@@@
state:
	pc 0
	memory:
		mem[ 0 ] 8454148
		mem[ 1 ] 4259841
		mem[ 2 ] 12648452
		mem[ 3 ] 25165824
		mem[ 4 ] 7
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 0
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 1
	memory:
		mem[ 0 ] 8454148
		mem[ 1 ] 4259841
		mem[ 2 ] 12648452
		mem[ 3 ] 25165824
		mem[ 4 ] 7
	registers:
		reg[ 0 ] 0
		reg[ 1 ] 7
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 2
	memory:
		mem[ 0 ] 8454148
		mem[ 1 ] 4259841
		mem[ 2 ] 12648452
		mem[ 3 ] 25165824
		mem[ 4 ] 7
	registers:
		reg[ 0 ] 0
		reg[ 1 ] -8
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state

@@@
state:
	pc 3
	memory:
		mem[ 0 ] 8454148
		mem[ 1 ] 4259841
		mem[ 2 ] 12648452
		mem[ 3 ] 25165824
		mem[ 4 ] -8
	registers:
		reg[ 0 ] 0
		reg[ 1 ] -8
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state
machine halted
total of 4 instructions executed
final state of machine:

@@@
state:
	pc 4
	memory:
		mem[ 0 ] 8454148
		mem[ 1 ] 4259841
		mem[ 2 ] 12648452
		mem[ 3 ] 25165824
		mem[ 4 ] -8
	registers:
		reg[ 0 ] 0
		reg[ 1 ] -8
		reg[ 2 ] 0
		reg[ 3 ] 0
		reg[ 4 ] 0
		reg[ 5 ] 0
		reg[ 6 ] 0
		reg[ 7 ] 0
end state
